client/src/parsec/login.ts

```diff
@@ -177,34 +180,61 @@ export async function getCurrentAvailableDevice(): Promise<Result<AvailableDevic
     }
     return { ok: true, value: currentAvailableDevice };
   } else {
-    return { ok: true, value: availableDevices[0] };
+    const device = availableDevices[0];
+    // device.ty = DeviceFileType.Keyring;
```

Comment:
what's this comment for?

Reply:
To be able to switch to keyring quickly for testing purposes, mostly for the UI. By uncommenting this, you can simulate a login with the keyring even on web with the mocks. It's not much, but since this is in the mock part, it doesn't really matter.